src/lib/mina.ts

```diff
@@ -394,11 +416,14 @@ let activeInstance: Mina = {
     ) {
       let account = Fetch.getCachedAccount(
         publicKey,
+        tokenId,
         Fetch.defaultGraphqlEndpoint
       );
       if (account === undefined)
         throw Error(
-          `getAccount: Could not find account for public key ${publicKey.toBase58()}. Either call Mina.setActiveInstance first or explicitly add the account with addCachedAccount`
+          `getAccount: Could not find account for public key ${publicKey.toBase58()} with the tokenId ${Ledger.fieldToBase58(
```

Comment:
same here (should this string be a constant stored somewhere instead of inlined all the time?)